tests/Rector/ClassMethod/MakeEntitySetterNullabilityInSyncWithPropertyRector/Fixture/many_to_one_default_setter_nullable.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Rector\ClassMethod\MakeEntitySetterNullabilityInSyncWithPropertyRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class ManyToOneDefaultSetterNullable
+{
+    /**
+     * @ORM\ManyToOne(targetEntity="DefaultEntity")
+     */
+    private $defaultEntity;
+
+    public function setDefaultEntity(?DefaultEntity $defaultEntity)
+    {
+        $this->defaultEntity = $defaultEntity;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Doctrine\Tests\Rector\ClassMethod\MakeEntitySetterNullabilityInSyncWithPropertyRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class ManyToOneDefaultSetterNullable
+{
+    /**
+     * @ORM\ManyToOne(targetEntity="DefaultEntity")
+     */
+    private $defaultEntity;
+
+    public function setDefaultEntity(?DefaultEntity $defaultEntity)
+    {
+        $this->defaultEntity = $defaultEntity;
+    }
+}
+
+?>
```

Comment:
It seems these 2 halfs are identical. 

If there is no change, the snippet must contain only top half.

Reply:
updated them